samples/tab-stage-view/csharp/TabInStageView/AppPackage/manifest.json

```diff
@@ -49,10 +49,43 @@
       "isNotificationOnly": false
     }
   ],
+  "composeExtensions": [
+    {
+      "botId": "<<MICROSOFT-APP-ID>>",
+      "messageHandlers": [
+        {
+          "type": "link",
+          "value": {
+            "domains": [
+              "<<DOMAIN-NAME>>"
+            ]
+          }
+        }
+      ],
+      "canUpdateConfiguration": true,
+      "commands": [
+        {
+          "id": "searchQuery",
+          "context": [ "compose", "commandBox" ],
+          "description": "Test command to run query",
+          "title": "Search",
+          "type": "query",
+          "parameters": [
+            {
+              "name": "searchQuery",
+              "title": "Search Query",
+              "description": "Your search query",
+              "inputType": "text"
+            }
+          ]
+        }
+      ]
+    }
+  ],
   "accentColor": "#60A18E",
   "permissions": [ "identity", "messageTeamMembers" ],
   "validDomains": [
-    "<<VALID DOMAIN>>",
+    "<<DOMAIN-NAME>>",
     "token.botframework.com"
```

Comment:
By default we generally add "ngrok.io"
Kindly check in other samples and update it on priority.